web-app/src/app/app.component.ts

```diff
@@ -68,6 +68,10 @@ export class AppComponent implements OnInit {
   }
 
   registerIcons(): void {
+    // observe changes and update public variable when changed
```

Comment:
Is this comment relevant anywhere in this function?

Reply:
I think it was auto-filled. I'll remove it